### DISCUSSION TITLE
Source Code of Mars Web Application

### DISCUSSION THREAD

**Discussion by simibac** [decentralized-identity/veramo#380] [Q&A] [answered]
Is the[ mars web application](https://mars.veramo.io/agent/discover) also open sourced?

**Comment by jasheal** [comment, marked as answer]
Mars is an unsupported experimental dashboard project for your agents. 

https://github.com/veramolabs/mars-app

Feel free to use it but it will be archived soon as we have an officially supported dashboard being open sourced next week.

**Reply by simibac** [reply]
Where can I find this project? Or has it not been release yet? How do I find out about other ongoing projects of the veramo suite like a mobile wallet etc?

**Reply by jasheal** [reply]
@simibac It hasn't been open sourced yet. I can add you to the preview if you would like to provide some feedback?

**Reply by simibac** [reply]
Sure I'd love that.

**Reply by jasheal** [reply]
Done. You will find the guide link to follow in the description.

**Reply by simibac** [reply]
@jasonhealy I have been experimenting with the agent-explorer for a while and created my own widget for issuing VC. I have some feedback and some general questions about the project. I think it would be easier to discuss in a call. Would you or any of the Veramo devs be willing to have a call with me in the next few days?

**Reply by jasheal** [reply]
@simibac Sounds good. Next week or so is pretty busy for us so if you have feedback/questions that you can share here in the meantime that would be very helpful.

**Reply by simibac** [reply]
Since you haven't open-sourced the agent-explorer repo yet, I wanted to ask if I can still make a **private** fork and use it as part of my master thesis for a PoC (currently forking is disabled). If so I could officially reference it in the paper. Or do you already have plans to open source it any time soon?

Regarding feedback:
- I like the widget/modules architecture. This makes it easy to build for custom needs 👌🏼
- Since the widget configuration is stored locally in the browser, renaming a module/widget leads to an invalid configuration. It might be helpful to point this out in the documentation. 
- The documentation for the agent-explorer is no longer accessible under https://veramo.io/docs/guides/agent_explore

**Reply by tsnyder-gs1us** [reply]
Any updates on when you will be releasing the new dashboard example?

**Reply by jasheal** [reply]
@simibac thanks for the feedback! A few things were moved around priority wise so this was delayed. We plan to open source it before the end of this month.

**Reply by intrinsicsolutionsdev** [reply]
Could i also get access please I'd love to have a look

**Reply by simibac** [reply]
It is open source now: https://github.com/veramolabs/agent-explorer
